python/tchannel/messages/base.py

```diff
@@ -25,3 +30,59 @@ def __str__(self):
             str(self.__class__.__name__),
             ", ".join(attrs)
         )
+
+    @staticmethod
+    def compute_checksum(checksum_type, args, csum=0):
```

Comment:
Given that these three related methods are static and don't really need to be in `BaseMessage`. Move to a separate module as top-level functions. `tchannel.messages.common` or `tchannel.checksum`.